seqr/views/pages/project_page.py

```diff
@@ -126,47 +126,22 @@ def _retrieve_families(cursor, project_guid, user):
     return families_by_guid
 
 
-def _retrieve_individuals(cursor, project_guid, user):
+def _retrieve_individuals(project_guid, user):
     """Retrieves individual-level metadata for the given project.
 
     Args:
-        cursor: connected database cursor that can be used to execute SQL queries.
         project_guid (string): project_guid
     Returns:
         dictionary: individuals_by_guid
     """
-    individuals_query = """
-        SELECT DISTINCT
-          p.guid AS project_guid,
-          f.guid AS family_guid,
-          i.guid AS individual_guid,
-          i.individual_id AS individual_individual_id,
-          i.display_name AS individual_display_name,
-          i.paternal_id AS individual_paternal_id,
-          i.maternal_id AS individual_maternal_id,
-          i.sex AS individual_sex,
-          i.affected AS individual_affected,
-          i.notes as individual_notes,
-          i.case_review_status AS individual_case_review_status,
-          i.case_review_status_last_modified_date AS individual_case_review_status_last_modified_date,
-          i.case_review_status_last_modified_by_id AS individual_case_review_status_last_modified_by,
-          i.case_review_discussion AS individual_case_review_discussion,
-          i.phenotips_patient_id AS individual_phenotips_patient_id,
-          i.phenotips_data AS individual_phenotips_data,
-          i.created_date AS individual_created_date,
-          i.last_modified_date AS individual_last_modified_date
-
-        FROM seqr_individual AS i
-          JOIN seqr_family AS f ON i.family_id=f.id
-          JOIN seqr_project AS p ON f.project_id=p.id
-        WHERE p.guid=%s
-    """.strip()
 
-    cursor.execute(individuals_query, [project_guid])
+    fields = Individual._meta.json_fields + Individual._meta.internal_json_fields + \
+             ['family__guid', 'case_review_status_last_modified_by__email']
```

Comment:
does `case_review_status_last_modified_by__email` work when case_review_status_last_modified_by is None?

Reply:
yes